pds4indextools/pds4_create_xml_index.py

```diff
@@ -1,69 +1,200 @@
 """
-XML Bundle Scraper
+PDS4 Indexing Tool
 
-This script scrapes XML files within specified directories, extracts information from
-user-defined XML elements, and generates a CSV index file. The script provides options
-for customizing the extraction process, such as specifying XPath headers, limiting
-search levels, and selecting elements to scrape.
+This script scrapes label files within specified directories, extracts information from
+user-defined XPaths/elements, and generates either an index file or a .txt file containing
+the XPaths available to the user. The script provides options for customizing the
+extraction process, such as:
+    - Specifying desired content by either limiting the results or requesting additional
+      file information.
+    - Sorting the resulting file by a user-specified value.
+    - Allowing for a user-made configuration file (.ini) for further custom content.
 
 Usage:
```

Comment:
Honestly I would just delete all of this from the docstring and tell the user to run `python pds4_create_xml_index.py --help`. That way you don't have to keep those two things in sync, and the help should always have all the needed information anyway.

Reply:
This comment is marked as checked off but all the usage data is still there.